packages/evm/src/evm.ts

```diff
@@ -129,6 +131,15 @@ export class EVM implements EVMInterface {
     return this._opcodes
   }
 
+  protected _isInitialized: boolean = false
```

Comment:
This initializer should be removed, the constructor is protected and to create an EVM you `await EVM.create`